cmd/kured/main.go

```diff
@@ -173,12 +173,20 @@ func main() {
 func flagCheck(cmd *cobra.Command, args []string) {
 	if slackHookURL != "" && notifyURL != "" {
 		log.Warnf("Cannot use both --notify-url and --slack-hook-url flags. Kured will use --notify-url flag only...")
-		slackHookURL = ""
-	}
-	if slackChannel != "" || slackHookURL != "" {
-		log.Warnf("slack-* flag(s) are being deprecated. Please use --notify-url flag instead.")
 	}
 
+	if slackHookURL != "" {
+		log.Warnf("Deprecated flag(s). Please use --notify-url flag instead.")
+		trataURL, err := url.Parse(slackHookURL)
```

Comment:
The more I think about this, the more I am inclined to say: "Don't ignore incorrect input. Instead crash to show the user has done something wrong". I know it's a change in our way of doing, but I would like us to think about it.

Which means, in this case:

- An error in parsing the slackHookUrl should return an error, and in caller function, we should handle that error by crashing.
- Side note: I think that the url could match a regex, and fail if it's not matching: (https://hooks.slack.com/services/(.*)/(.*)/(.*)), the result don't need to be tested for length, as the expectations are there. Again, returning an error would be more explicit.
- Alternative to previous idea, assign a slice after split(trim (path)). This would make L184 and L187 clearer.

Reply:
I am not so sure regarding the crash if slack hook url is incorrect formatted. @dholbach, @ckotzbauer, @jackfrancis what say you?

Reply:
I thought it htrough, I think the regexp.Mustcompile() thing would be obsolete because url.Parse() from net/url is already doing this. It will throw an error and the user will see it in the log. Question still remains: log.Printf or log.Fatalf ? 
Also, if there is "/" at the end of the URL, shoutrrr will throw an error which will be exposed in the log as well.